common-npm-packages/packaging-common-v3/universal/ClientToolRunner.ts

```diff
@@ -0,0 +1,68 @@
+let fs = require("fs");
+let os = require("os");
+import * as tl from "azure-pipelines-task-lib";
+import child = require("child_process");
+import stream = require("stream");
+import {IExecOptions, IExecSyncResult} from "azure-pipelines-task-lib/toolrunner";
+
+export interface IClientToolOptions {
+    clientToolFilePath: string;
```

Comment:
This feels very specific to Symbol. I checked the Original PR and looks like this is being used only to send data from 1 function to another, as an object, rather than individual strings. imo, we should remove this interface in generic file and have it in symbol specific implementation.